posthog/api/tagged_item.py

```diff
@@ -7,38 +7,50 @@
 
 class TaggedItemSerializerMixin(serializers.Serializer):
     """
-    Serializer mixin that aliases tags -> global_tags and determines appropriate response depending on license.
+    Serializer mixin that resolves appropriate response for tags depending on license.
     """
 
     tags = WritableSerializerMethodField(required=False)
 
-    def get_tags(self, obj):
-        if (
+    def is_licensed(self):
+        return (
             "request" in self.context
             and not self.context["request"].user.is_anonymous
-            and self.context["request"].user.organization.is_feature_available(AvailableFeature.INGESTION_TAXONOMY)
-        ):
+            and self.context["request"].user.organization.is_feature_available(AvailableFeature.DASHBOARD_COLLABORATION)
```

Comment:
I might've been misleading in the last review - sorry. I was more thinking that we should add a new feature called `TAGGING`. It feels like this feature spans both `INGESTION_TAXONOMY` and `DASHBOARD_COLLABORATION`